java-agent/instrumentation/spring-webflux-5/src/main/java8/io/opentelemetry/auto/instrumentation/springwebflux/client/TracingClientResponseMono.java

```diff
@@ -28,26 +25,33 @@ public TracingClientResponseMono(
     this.exchangeFunction = exchangeFunction;
   }
 
+  private static Span getCurrentSpan() {
+    return TRACER.getCurrentSpan();
+  }
+
```

Comment:
Not needed if `TRACER::getCurrentSpan` is used below

Reply:
I was planning for that method to do more but ended up not needing it...